sdk/Sdk.Generators/Abstractions/GeneratorFunctionMetadata.cs

```diff
@@ -21,6 +21,6 @@ internal class GeneratorFunctionMetadata
 
         public IDictionary<string, object> Properties { get; set; } = new Dictionary<string, object>();
 
-        public IList<IDictionary<string, string>> RawBindings { get; set; } = new List<IDictionary<string, string>>(); // List of <propertyName, propertyValue> bindings.
+        public IList<IDictionary<string, object>> RawBindings { get; set; } = new List<IDictionary<string, object>>(); // List of <propertyName, propertyValue> bindings.
```

Comment:
`string` -> `object` change is due to switching to JSON serialization. Previously, arrays were stored as a string that would be written right to the generated file, but now we need the actual array object.